src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -2115,6 +2115,13 @@ private static bool IsUserDefinedTrueOrFalse(BoundUnaryOperator @operator)
                     type = null;
                     nullability = new NullabilityInfo(CodeAnalysis.NullableAnnotation.NotAnnotated, CodeAnalysis.NullableFlowState.NotNull);
                 }
+                else if (highestBoundExpr is BoundConvertedSwitchExpression e)
```

Comment:
>highestBoundExpr [](start = 25, length = 16)

Is there a particular reason to use ```highestBoundExpr``` here instead of```boundExpr```?

Reply:
I cannot think of a way that using `boundExpr` would make a difference and yet be correct.

---
In reply to: [438931621](https://github.com/dotnet/roslyn/pull/45069#discussion_r438931621) [](ancestors = 438931621)

Reply:
> I cannot think of a way that using boundExpr would make a difference and yet be correct.

Could you elaborate why that would be incorrect?

---
In reply to: [439055465](https://github.com/dotnet/roslyn/pull/45069#discussion_r439055465) [](ancestors = 439055465,438931621)

Reply:
Please give an example of how it would make a difference and I will describe my concern with that difference.

---
In reply to: [439105960](https://github.com/dotnet/roslyn/pull/45069#discussion_r439105960) [](ancestors = 439105960,439055465,438931621)